app/views/devise/shared/_login_form.html.slim

```diff
@@ -1,24 +1,23 @@
 = form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
-  form.form-horizontal[novalidate]
-    = devise_error_messages!
-    fieldset.form-group.position-relative.has-icon-left
-      - f.label :email
-      = f.email_field :email, id: :user_email ,
+  = devise_error_messages!
+  fieldset.form-group.position-relative.has-icon-left
+    - f.label :email
```

Comment:
w/o adding to output buffer?

Reply:
Roman mean why you have "-" instead of "="